code/modules/mob/living/simple_animal/hostile/glockroach.dm

```diff
@@ -0,0 +1,69 @@
+/obj/projectile/glockroachbullet
+	damage = 10 //same damage as a hivebot
+	damage_type = BRUTE
+
+/obj/item/ammo_casing/glockroach
+	name = "0.9mm bullet casing"
+	desc = "A... 0.9mm bullet casing? What?"
+	projectile_type = /obj/projectile/glockroachbullet
+
+/mob/living/simple_animal/hostile/glockroach //copypasted from cockroach.dm so i could use the shooting code in hostile.dm
+	name = "glockroach"
+	desc = "HOLY SHIT, THAT COCKROACH HAS A GUN!"
+	icon_state = "glockroach"
+	icon_dead = "cockroach"
+	health = 1
+	maxHealth = 1
+	turns_per_move = 5
+	loot = list(/obj/effect/decal/cleanable/insectguts)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 270
+	maxbodytemp = INFINITY
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	mob_size = MOB_SIZE_TINY
+	mob_biotypes = MOB_ORGANIC|MOB_BUG
+	response_disarm_continuous = "shoos"
+	response_disarm_simple = "shoo"
+	response_harm_continuous = "splats"
+	response_harm_simple = "splat"
+	speak_emote = list("chitters")
+	density = FALSE
+	ventcrawler = FALSE //a ventcrawling mob with a gun is probably a bad idea
```

Comment:
coward

Reply:
although the idea of someone getting ambushed and gunned down by a roaming gang of cockroaches is a very entertaining mental image, it's probably a bad idea to allow it just so its not impossible to kill

Reply:
however, at the same time, it DOES only have 1 health and dies if you step on it

Reply:
I say leave the vent crawl because it only does 10 damage per shot anyway, it needs to hit you ten times to crit